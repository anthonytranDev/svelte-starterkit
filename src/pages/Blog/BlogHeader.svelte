<style>
  .blog-description {
    color: #686868;
    font-style: italic;
  }
</style>

<div data-testid="BlogHeader">
  <h1>Blog</h1>
  <p class="blog-description">Latest news</p>
</div>
