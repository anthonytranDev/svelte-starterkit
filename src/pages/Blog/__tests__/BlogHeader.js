import "@testing-library/jest-dom/extend-expect"
import { render } from "@testing-library/svelte"

import BlogHeader from "../BlogHeader.svelte"

describe("BlogHeader", () => {
  it("should render", () => {
    const { getByTestId } = render(BlogHeader)

    expect(getByTestId("BlogHeader")).toMatchSnapshot()
  })
})
