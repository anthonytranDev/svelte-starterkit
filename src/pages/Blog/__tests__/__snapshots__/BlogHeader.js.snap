// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BlogHeader should render 1`] = `
<div
  data-testid="BlogHeader"
>
  <h1>
    Blog
  </h1>
   
  <p
    class="blog-description svelte-y06stb"
  >
    Latest news
  </p>
</div>
`;
